src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.section {
```

Comment:
what is section? it's separate BEM block? if yes - these styles should be in separate css file